server/request.c

```diff
@@ -157,24 +160,93 @@ AP_DECLARE(int) ap_some_authn_required(request_rec *r)
     return rv;
 }
 
+static int walk_location_and_if(request_rec *r)
+{
+    int access_status;
+
+    if ((access_status = ap_location_walk(r))) {
+        return access_status;
+    }
+    if ((access_status = ap_if_walk(r))) {
+        return access_status;
+    }
+
+    /* Don't set per-dir loglevel if LogLevelOverride is set */
+    if (!r->connection->log) {
```

Comment:
That `if (!r->connection->log)` is from https://github.com/apache/httpd/commit/d04d01c5891a1f0f8fadd5e16f1752b915d2153f which never made it to 2.4.x (AFAICT).
For now it probably should be simply:
```
    {
        core_dir_config *d...
        ...
    }
```
in 2.4.x (with the unconditional block which aligns with trunk still).

Reply:
OK I will fix that thanks.

Reply:
fixed.